app/src/main/res/values/strings.xml

```diff
@@ -376,6 +376,11 @@ work. If not, see <https://creativecommons.org/publicdomain/zero/1.0/>.
     <string name="expand_collapse">Expand / Collapse</string>
     <string name="essential">Essential</string>
     <string name="append_human_readable_uuid_to_new_tasks">Append human readable UUID to new tasks</string>
+    <string name="document_password">Password for file encryption.</string>
+    <string name="no_password_found_warning"><![CDATA[No password found, can\'t en-/decrypt.\nGo to \'Settings -> General -> Password\' to set a password.]]></string>
+    <string name="encrypt_file_content">encrypt file content</string>
+    <string name="failed_to_encrypt_document">Couldn\'t encrypt_message. The document wasn\'t stored.</string>
```

Comment:
could_not_encrypt_file_content_the_file_was_not_saved
Could not encrypt file content. The file was not saved